default.json

```diff
@@ -46,7 +47,22 @@
   ],
   "packageRules": [
     {
-      "description": "Automatically merge minor and patch updates for Microsoft dependencies",
+      "groupName": "Allow Workleap's NuGet and NPM packages to be updated immediately",
+      "minimumReleaseAge": "0 days",
+      "matchPackageNames": [
+        "/^[wW]orkleap\\./",
+        "/^[oO]ffice[vV]ibe\\./",
+        "/^[sS]share[gG]ate\\./",
+        "/^@workleap",
+        "/^@sharegate",
+        "/^@officevibe",
+        "/^@hopper-ui/",
+        "/^@orbit-ui/",
+        "/^@squide/"

```

Comment:
I looked for our public and internal packages/dependencies on GitHub and Azure DevOps